hal/src/hal.cpp

```diff
@@ -14,6 +15,16 @@ RPYBUILD_PYBIND11_MODULE(m) {
   m.attr("__hal_simulation__") = true;
 #endif
 
+  // Add this manually so it can be used from SimValue
+  py::enum_<HAL_Type>(m, "Type")
```

Comment:
Any reason to not autogen this from Value.h?

Reply:
Then would need to exclude all of the other things. Didn't seem worth it.